assume/common/outputs.py

```diff
@@ -40,18 +40,18 @@ def __init__(
         start: datetime,
         end: datetime,
         db_engine=None,
-        export_csv_path: str = "",
-        save_frequency_hours: int = 24,
+        export_csv_path: str = None,
+        save_frequency_hours: int = None,
         learning_mode: bool = False,
     ):
         super().__init__()
 
         # store needed date
         self.simulation_id = simulation_id
-        self.save_frequency_hours = save_frequency_hours
+        self.save_frequency_hours = save_frequency_hours or (end - start).days * 24
```

Comment:
When would you want to disable saving results in given intervals?
Stating "same_process=False" should fix consuming performance, by using a separate process

The remaining data will always be written in `on_stop` of the outputs role

Reply:
as of right now this saving each day consumes a big chunk of time as multithread is not working. We should have this until second thread works. Also, sometime you really don't need all that data every day of the run...

Reply:
I believe this is also not because of the db, but the csv files, those require a lot of time